src/renderer/Dataframe.js

```diff
@@ -374,21 +374,21 @@ export default class Dataframe {
 
 // Returns true if p is inside the triangle or on a triangle's edge, false otherwise
 // Parameters in {x: 0, y:0} form
-export function pointInTriangle(p, v1, v2, v3) {
+export function _pointInTriangle(p, v1, v2, v3) {
```

Comment:
Hey, it is not very cool to mix changes of this kind with the changes of moving files between folders. CRs for moving files are almost completely overlooked, in fact, I saw this on `master` after the merge. Please, do not mix this kind of refactor with file/folder changes.